### DISCUSSION TITLE
posts/cpp

### DISCUSSION THREAD

**Discussion by giscus** [shilinlee/blog#41] [Comments]
# posts/cpp

C++入门 简介 C++是一种通用、静态类型、综合高低级语言的编程语言。支持面向对象、泛型编程。始于1979年，C的超集。 环境设置 C++环境设置需文本编辑器和编译器。常用编辑器有Notepad、vim；编译器以GCC为例，Linux可用g++，Windows需安装MinGW。 基本语法 C++程序由类、对象、方法和即时变量组成。类是描述对象行为和状...

https://shilinlee.com/posts/cpp.html

**Comment by shilinlee** [comment]
快速入门C++
